controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -453,11 +460,16 @@ public String getScopeName() {
                                         Throwable ex = ExceptionHelpers.getRealException(e);
                                         if (ex instanceof StoreException.DataNotEmptyException) {
                                             // cant delete as there are transactions still running under epoch node
+                                            log.debug("stream {}/{} epoch {} not empty", scope, name, epoch);
                                             result.complete(false);
                                         } else {
+                                            log.warn("stream {}/{} deleting epoch {} threw exception {}", scope, name, epoch, ex.getClass().getName());
```

Comment:
We could use this api `org.slf4j.Logger#warn(java.lang.String, java.lang.Throwable)`?